sharding/indexHashedNodesCoordinator.go

```diff
@@ -27,6 +34,18 @@ func NewIndexHashedNodesCoordinator(arguments ArgNodesCoordinator) (*indexHashed
 		return nil, err
 	}
 
+	var cache NodesCoordinatorCache
+
+	if arguments.ConsensusGroupCache == nil {
+		cache, err = lrucache.NewCache(defaultCacheSize)
+	} else {
+		cache = arguments.ConsensusGroupCache
```

Comment:
it should be directly with the cache as a must, not as a choice. no need to create new cache here.

Reply:
yes, removed initialization